tests/rt.conf

```diff
@@ -21,10 +21,18 @@
 
 ### Intel Tests ###
 ### S2S tests ###
-COMPILE | s2swa_32bit  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 |          | fv3 |
+COMPILE | s2swa_32bit  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_v17_coupled_p8_ugwpv1 |          | fv3 |
 RUN | cpld_control_p8_mixedmode                         |    - noaacloud                     | baseline |
-RUN | cpld_control_gfsv17                               |                                    | baseline |
-RUN | cpld_control_gfsv17_iau                           |                                    | baseline | cpld_control_gfsv17
+
+#GFS
+COMPILE | s2swa_32bit_pdlib  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON | + hera orion cheyenne wcoss2 acorn | fv3 |
+RUN | cpld_control_gfsv17                               | + hera orion cheyenne wcoss2 acorn | baseline |
+RUN | cpld_restart_gfsv17                               | + hera orion cheyenne wcoss2 acorn |          | cpld_control_gfsv17 
+RUN | cpld_mpi_gfsv17                                   | + hera orion cheyenne wcoss2 acorn |          | cpld_control_gfsv17
+RUN | cpld_control_gfsv17_iau                           | + hera orion cheyenne wcoss2 acorn | baseline | 
```

Comment:
cpld_control_gfsv17_iau  test has dependency on cpld_control_gfsv17, not cpld_mpi_gfsv17.

Reply:
Also is there a reason the tests can't run on gaea and jet?

Reply:
The PDLIB tests were not activated on Gaea or Jet.  Perahps @DeniseWorthen can provide input on that. 

In terms of dependency is this just the order that you would like the cpld_control_gfsv17_iau test to appear in this list?

Reply:
@JessicaMeixner-NOAA I believe the initial restriction on where we run the PDLIB cases had to do w/ where the required lib (might have been metis/parmetis at that time) had been installed.

Reply:
That makes a lot of sense @DeniseWorthen, if scotch is installed everywhere we could do the -noaacloud option instead for these tests and keep it consistent with options before.  We might also need to then make sure input is moved over as well.  I know it's on Acorn which I added and I could look for it on gaea, but I don't have jet access to look there.  

Let me know what the preference is and I'll add it to my branch I'm testing to push to Mikes branch here.